multy_core/src/bitcoin/bitcoin_account.cpp

```diff
@@ -117,6 +117,65 @@ class BitcoinP2PKHAccount : public BitcoinAccount
     }
 };
 
+class BitcoinP2SHWPKHAccount : public BitcoinAccount
+{
+public:
+    using BitcoinAccount::BitcoinAccount;
+
+    std::string get_address() const override
+    {
+
+        // P2SHPKH address generated from public key.
+        unsigned char pub_hash[HASH160_LEN + 1] = {'\0'};
+
+        // 1 - Take the corresponding public key generated with it (33 or 65
+        // bytes)
+        PublicKeyPtr public_key(m_private_key->make_public_key());
+        BinaryData key_data = public_key->get_content();
+
+        // 2 - Perform SHA-256 hashing on the public key
```

Comment:
Please move the comments above corresponding lines of the code, not just above the block.